src/main/kotlin/lotto/LottoController.kt

```diff
@@ -1,24 +1,56 @@
 package lotto
 
-import lotto.domain.LottoResult
-import lotto.domain.LottoStore
+import lotto.domain.AutoStrategy
+import lotto.domain.LottoGame
+import lotto.domain.LottoNumber
+import lotto.domain.LottoTicket
+import lotto.domain.Lottoes
+import lotto.domain.ManualStrategy
+import lotto.domain.Money
 import lotto.domain.WinningLotto
 import lotto.ui.InputView
 import lotto.ui.OutputView
 
+private val inputView = InputView()
+private val outputView = OutputView()
+
 fun main() {
-    val inputView = InputView()
-    val outputView = OutputView()
-    val lottoStore = LottoStore()
-    val lottoResult = LottoResult()
+    val money = Money(inputView.inputMoney())
+    val numberOfManual = inputView.inputNumberOfManual()
+    val lottoGame = LottoGame(money)
+
+    val manualLottoes = inputManualLottoes(numberOfManual, lottoGame)
+    val autoLottoes = lottoGame.purchaseLottoes(AutoStrategy())
+
+    outputView.printPurchasedLottoes(manualLottoes, autoLottoes)
+    val universalLottoes = Lottoes(manualLottoes.toList() + autoLottoes.toList())
```

Comment:
코틀린에서는 정해진 이름의 함수를 오버로딩함으로써 표준 수학 연산자를 오버로딩할 수 있습니다. 아래의 글이 도움 될 거예요.
https://kotlinlang.org/docs/operator-overloading.html

```suggestion
    val universalLottoes = manualLottoes + autoLottoes
```